qabelbox/src/main/java/de/qabel/qabelbox/fragments/CreateIdentityEditTextFragment.java

```diff
@@ -44,21 +56,49 @@ public static CreateIdentityEditTextFragment newInstance(int messageId, int edit
     public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle
             savedInstanceState) {
 
-        View view = inflater.inflate(R.layout.fragment_create_identity_edittext, container, false);
+        final View view = inflater.inflate(R.layout.fragment_create_identity_edittext, container, false);
 
         TextView tvMessage = ((TextView) view.findViewById(R.id.tv_message));
-        editText = (EditText) view.findViewById(R.id.et_name);
         tvMessage.setText(mMessageId);
-        editText.setHint(mEditTextHintId);
+
+        editText = (EditText) view.findViewById(R.id.et_name);
+        if (value != null) {
+            editText.setText(value);
+        }
+        if (!optionalValue) {
+            editText.setHint(mEditTextHintId);
+        } else {
+            editText.setHint(getString(mEditTextHintId) + " (" + getString(R.string.optional) + ")");
+        }
         if (inputType != null) {
             editText.setInputType(InputType.TYPE_CLASS_TEXT | inputType);
         }
+        editText.setImeOptions(EditorInfo.IME_ACTION_NEXT);
+        editText.setOnEditorActionListener(new TextView.OnEditorActionListener() {
+            @Override
+            public boolean onEditorAction(TextView v, int actionId, KeyEvent event) {
+                if (actionId == EditorInfo.IME_ACTION_NEXT) {
+                    Activity activity = getActivity();
+                    if (activity != null && activity instanceof BaseWizardActivity) {
+                        ((BaseWizardActivity) activity).handleNextClick();
+                        return true;
+                    }
+                }
+                return false;
+            }
+        });
```

Comment:
Which tests address these changes? (both the optionalValue and the onEditorActionListener)